src/status_im/contexts/shell/activity_center/notification/common/view.cljs

```diff
@@ -103,47 +97,54 @@
   (rf/dispatch [:activity-center.notifications/delete (:id notification)]))
 
 (defn swipeable
-  [_]
-  (let [swipeable-ref (atom nil)]
-    (fn [{:keys [active-swipeable
-                 extra-fn
-                 left-button
-                 left-on-press
-                 right-button
-                 right-on-press]}
-         & children]
-      (into
-       [gesture/swipeable
-        (merge
-         {:ref                      #(reset! swipeable-ref %)
-          :accessibility-label      :notification-swipeable
-          :friction                 2
-          :on-swipeable-will-open   (close-active-swipeable active-swipeable swipeable-ref)
-          :children-container-style {:padding-horizontal 20}}
-         (when left-button
-           {:overshoot-left      false
-            :left-threshold      style/swipe-action-width
-            :render-left-actions (render-swipe-action
-                                  {:active-swipeable active-swipeable
-                                   :extra-fn extra-fn
-                                   :interpolation-opacity style/left-swipe-opacity-interpolation-js
-                                   :interpolation-translate-x
-                                   style/left-swipe-translate-x-interpolation-js
-                                   :on-press left-on-press
-                                   :swipe-button left-button
-                                   :swipeable-ref swipeable-ref
-                                   :style {:left 8}})})
-         (when right-button
-           {:overshoot-right      false
-            :right-threshold      style/swipe-action-width
-            :render-right-actions (render-swipe-action
-                                   {:active-swipeable active-swipeable
-                                    :extra-fn extra-fn
-                                    :interpolation-opacity style/right-swipe-opacity-interpolation-js
-                                    :interpolation-translate-x
-                                    style/right-swipe-translate-x-interpolation-js
-                                    :on-press right-on-press
-                                    :swipe-button right-button
-                                    :swipeable-ref swipeable-ref
-                                    :style {:right -8}})}))]
-       children))))
+  [{:keys [extra-fn
+           left-button
+           left-on-press
+           right-button
+           right-on-press]}
+   child]
```

Comment:
Make the component accept only one child instead of the slower `children` + `into` combo.